cmd/mongodb/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	"github.com/mainflux/mainflux"
+	log "github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/writers"
+	mongodb "github.com/mainflux/mainflux/writers/mongodb"
+	nats "github.com/nats-io/go-nats"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	name                = "mongodb-writer"
+	defNatsURL          = nats.DefaultURL
+	defPort             = "8180"
+	defDBName           = "mainflux"
+	defDBCollection     = "mainflux"
+	defDBHost           = "localhost"
+	defDBPort           = "8087"
+	defDBUser           = "mainflux"
+	defDBPass           = "mainflux"
+	defDBConnectTimeout = 5000
+	defDBSocketTimeout  = 5000
+
+	envNatsURL          = "MF_NATS_URL"
+	envPort             = "MF_MONGODB_WRITER_PORT"
+	envDBName           = "MF_MONGODB_DB_NAME"
+	envDBCollection     = "MF_MONGODB_DB_COLLECTION"
+	envDBHost           = "MF_MONGODB_DB_HOST"
+	envDBPort           = "MF_MONGODB_DB_PORT"
+	envDBUser           = "MF_MONGODB_DB_USER"
+	envDBPass           = "MF_MONGODB_DB_PASS"
+	envDBConnectTimeout = "MF_MONGODB_DB_CONNECTION_TIMEOUT"
```

Comment:
Let's omit these two timeout settings, and just use the constant values for now. I don't see much value of having a configurable connection/socket timeouts.